skimage/segmentation/_expand_labels.py

```diff
@@ -0,0 +1,107 @@
+""" 
+expand_labels is  inspired by code in CellProfiler, 
+code licensed under BSD-3 licenses.
+Website: http://www.cellprofiler.org
+
+Copyright (c) 2020 Broad Institute
+All rights reserved.
+
+Original author/s: Cellprofiler team 
+"""
+
+import numpy as np
+from scipy.ndimage import distance_transform_edt
+
+
+def expand_labels(label_image, distance):
+    """Expand labels in label image by ``distance`` pixels without overlapping.
+
+    Given a label image, each label is grown by up to distance pixels.
```

Comment:
What does grow mean exactly?  That the border expands "outwards" by up to N pixels?  I don't think the meaning here is that you grow it only by ``distance`` pixels.

Reply:
thanks, I will try and reword this